dist/less/order-service.less

```diff
@@ -170,17 +175,24 @@
       }
     }
   }
+  .review-message {
+    padding-left: 10px;
+  }
+  .review-status {
+    display: flex;
+    font-size: 16px;
+
+    .pficon {
+      margin-top: 3px;
+    }
+  }
 }
 
-.wizard-pf-contents {
+.order-service-wizard-step.wizard-pf-main {
   @media (min-width: @screen-sm-min) {
-    display: flex;
-    min-height: 100%;
-    padding-bottom: (@grid-gutter-width / 2);
-    padding-top: (@grid-gutter-width / 2);
-    .scroll-shadows-horizontal-covers();
-    _::-webkit-full-page-media, _:future, :root & { // only target Safari
-      background: none;
-    }
+    height: calc(@order-service-page-height + @grid-gutter-width);
```

Comment:
> dozens of parameters are within the advanced options, or are regular options? I thought that you and Jessica asked us to get customer feedback on how many of these options are actually specified upfront, implying that we think we are offering too many parameters. Did i misunderstand that?

There is no advanced options page for service instance parameters (yet anyway). For context, there are three different forms we show in the overlay from the landing page catalog currently:

1. Builder image forms. We have control over what's shown here. This is the from from #27 that we wanted customer feedback on.
2. Template forms. We **don't** control what the parameters are.
3. Service instance forms (provisioning services from the catalog). We **don't** control what the parameters are.

For (1) and (2), we've reused existing pages in the web console to add the advanced options link. We have not done that for (3), and we don't have an existing page to reuse.

(2) will go away after tech preview when we enable the template broker, so the templates will then show up like other service instances (3) and we'll have to revisit how we deal with parameters.

In short, we've (sort of) solved it for 2 of the 3 cases, but I'm not sure we have a good solution for service instance parameters. If we need to create a standalone provision service page for advanced options, we should open a Trello card because that's another chunk of work that isn't being tracked.